mmedit/models/components/stylegan2/common.py

```diff
@@ -0,0 +1,87 @@
+import torch
+
+
+def get_module_device(module):
+    """Get the device of a module.
+
+    Args:
+        module (nn.Module): A module contains the parameters.
+
+    Returns:
+        torch.device: The device of the module.
+    """
+    try:
+        next(module.parameters())
+    except StopIteration:
+        raise ValueError('The input module should contain parameters.')
+
+    if next(module.parameters()).is_cuda:
+        return next(module.parameters()).get_device()
+    else:
+        return torch.device('cpu')
+
+
+@torch.no_grad()
+def get_mean_latent(generator, num_samples=4096, bs_per_repeat=1024):
```

Comment:
Need docstring

Reply:
@nbei Could you help with that? I have copied the docstring of `get_mean_latent` from mmgeneration, but there is no docstring for `style_mixing` in mmgeneration.